algorithm/cgal/fromPolygonSoup.js

```diff
@@ -24,6 +24,13 @@ export const fromPolygonSoup = (
       cornerThreshold,
     })}`
   );
+  if (cornerThreshold === 0) {
+    try {
```

Comment:
What does this do?

Reply:
Skips simplification if all corners should be preserved.

Reply:
but why does it throw an error and then catch it?

Reply:
Ah, it gives you a stack trace to figure out how you got there. :)

I was trying to figure out what wasn't propagating the threshold through properly.